src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.csproj

```diff
@@ -21,4 +21,10 @@
     <ProjectReference Include="$(SharedSourceRoot)\Microsoft.AspNetCore.Razor.Utilities.Shared\Microsoft.AspNetCore.Razor.Utilities.Shared.csproj" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <AdditionalFiles Include="PublicAPI.Shipped.txt" />
+    <AdditionalFiles Include="PublicAPI.Unshipped.txt" />
+    <AdditionalFiles Include="InternalAPI.Shipped.txt" />
+    <AdditionalFiles Include="InternalAPI.Unshipped.txt" />
```

Comment:
😮 I had no idea the API analyzers support internal APIs. Will add that to ExternalAccess legacy editor.

Reply:
'twas written by our very own @333fred and is used in Roslyn, but sorely lacking in documentation, and copying Roslyn's use was less than fruitful, but we got there!